storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/operations/delete/PreparedDeleteObjectTest.java

```diff
@@ -100,7 +101,7 @@ public void shouldThrowExceptionIfNoTypeMappingWasFoundWithoutAffectingContentPr
             try {
                 preparedDelete.executeAsBlocking();
                 fail();
-            } catch (IllegalStateException expected) {
+            } catch (StorIOException expected) {
```

Comment:
Please add assertion for exception cause type and message, we must be sure that it's really expected exception.